src/main/java/com/typee/model/ModelManager.java

```diff
@@ -39,7 +39,7 @@ public ModelManager(ReadOnlyAddressBook addressBook, ReadOnlyUserPrefs userPrefs
         this.addressBook = new AddressBook(addressBook);
         this.statedAppointmentList = new StatedAddressBook(addressBook);
         this.userPrefs = new UserPrefs(userPrefs);
-        filteredPersons = new FilteredList<>(this.addressBook.getPersonList());
+        filteredPersons = new FilteredList<>(this.statedAppointmentList.getPersonList());
```

Comment:
perhaps can refactor the attribute filteredPersons to filteredAppointments or filteredEngagements. To reduce confusions in the future.